src/GenericContainer/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+# set target name
+add_bipedal_component(
+    NAME                   GenericContainer
+    PUBLIC_HEADERS         include/BipedalLocomotionControllers/GenericContainer.h include/BipedalLocomotionControllers/TemplateHelpers.h
```

Comment:
Why not in `include/BipedalLocomotionControllers/GenericContainer/ *`? The files will be installed in 
https://github.com/dic-iit/bipedal-locomotion-controllers/blob/e055599aa3cbe6bc3f4976a53c358694a5c0970a/cmake/AddBipedalLocomotionControllersComponent.cmake#L87

Reply:
I did not notice, I'll fix it.

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-controllers/pull/29/commits/2fd26da1dcaac5814f255c8e7d2ba2cea0dcae15